website/pages/blog/azure-cost-optimization-with-cloudquery.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Correlating Costs to Resources for Azure Cost Optimization with CloudQuery
+tag: tutorial
+date: 2022/12/23
+description: >-
+  This tutorial will show how to correlate between Azure Cost Analysis data and CloudQuery to optimize cost
+author: kemal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+Optimizing cloud costs is a never-ending task. There are many ways to do it, and many tools to help you do it. One of the most important things to do is to correlate your cloud costs with your cloud infrastructure. This tutorial will show how to correlate between Azure Cost Analysis data and CloudQuery to optimize cost.
+
+## Prerequisites
+
+- [Azure Source Plugin](https://www.cloudquery.io/docs/plugins/sources/azure/overview)
+- [PostgreSQL Destination Plugin](https://www.cloudquery.io/docs/plugins/destinations/postgresql/overview)
+- A running PostgreSQL instance (can be local or remote)
+
+## Create a Cost Analysis Report in Azure
+
+First, we need to create a _Cost Analysis Report_ in Azure. To do that, go to the Cost Management section in the Azure Portal and click on the [Cost Analysis](https://portal.azure.com/#view/Microsoft_Azure_CostManagement/Menu/~/costanalysis/openedBy/AzurePortal) tab.
+
+![Azure Cost Analysis Start Page](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report.png)
+
+Then, click on the _Cost by Resource_ button.
+
+![Azure Cost By Resource Menu](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-menu.png)
+
+Here you can make adjustments to your report. Make sure to select the time range you want to analyze. For this tutorial, we'll leave this at the default of current billing period. Make sure the _Group By_ option is set to **Resource**.
+
+![Azure Cost By Resource Report](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report-annotated.png)
+
+Use the _Save as_ button to save this as a new report view. Give it a name and click _Save_.
+
+![Azure Cost By Resource Report Save Button](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report-save.png)
+
+From now on every time we run a `cloudquery sync` command, the data from this report will be synced to PostgreSQL database. 
```

Comment:
```suggestion
Now we can configure CloudQuery so each time we run `cloudquery sync` the data from this report will be synced to a PostgreSQL database.
```

Not sure, but technically isn't `From now on every time` is only correct after following the next steps?

Reply:
Yes better, thanks :)